core/tests/test_deconvolution.py

```diff
@@ -24,7 +24,7 @@
 try:
     import pycudadecon._libwrap
     cuda_decon_available = True
-except FileNotFoundError:
+except (FileNotFoundError, ModuleNotFoundError):
```

Comment:
napari GUI will have a pop up saying you need to install pycudadecon. 
Not sure if the cli shows that warning

Reply:
This change is just for the sake of the tests, so they don't fail if it's not installed